src/index.ts

```diff
@@ -153,6 +153,8 @@ const graphqlServer = graphqlHTTP((req, res, params) => {
   const xImpersonateUserID = req.headers["x-impersonate-user-id"] as
     | string
     | undefined
+  const referer = req.headers["referer"] as string | undefined
+  const isCMSRequest = !!referer?.includes("cms")
```

Comment:
We should maybe update Volt's Relay middleware to propagate a `X-CMS-Request: 1` header or something just to 100% be able to tell this is a CMS request from within MP.